File: Solution_Practice_I/python/prob2_11.py
while True:
    print("Ingrese la cantidad de entradas: ")
    quantity = int(input())
    if 0 <= quantity < 15:
        break
    
for i in range(quantity):
    while True:
        while True:
            print("Ingrese 2 números: → (a b)")
            char_value = input().strip()
            values = char_value.split()
            if len(values) == 2:
                break
        a, b = int(values[0]), int(values[1])
        
        if (a if a>0 else a*-1) < 1000000001 and (b if b>0 else b*-1) < 1000000001:
            break
        
    if a > b:
        print(">")
    elif a < b:
        print("<")
    else:
        print("=")

File: Solution_Practice_I/java/prob2_11.java
import java.util.Scanner;
public class prob2_11 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int quantity,a,b;
        do {
            System.out.println("Ingrese la cantidad de entradas");
            quantity=input.nextInt();
            input.nextLine();
        } while (quantity>=15 || quantity<0);
        for (int i=1; i<=quantity; i++) {
            String charValue;
            String[] values;
            do {
                do {
                    System.out.println("Ingrese 2 numeros");
                    charValue=input.nextLine();
                    charValue=charValue.trim();
                    values = charValue.split(" ");
                } while (values.length != 2);
                a=Integer.parseInt(values[0]);
                b=Integer.parseInt(values[1]);
                
            } while ((a>0?a:a*-1)>=1000000001 || (b>0?b:b*-1)>=1000000001);
            if (a>b) System.out.println(">");
            else if (a<b) System.out.println("<");
            else System.err.println("=");
        }

        input.close();
    }
}